recipes/libftdi/1.x/test_package/CMakeLists.txt

```
@@ -1,6 +1,6 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.1)
 
-project(test_package LANGUAGES CXX)
+project(test_package)
```

The actual test package, unlike the V1 test package, should set the language here. `CXX` since C++ is in use.

```suggestion
project(test_package LANGUAGES CXX)
```

---

Sorry but C++ is used in both testpackages. The source code behinde both projects are the same. So I cant see the diffrence. So please merge the pull-request.